src/data/actions/assets.js

```diff
@@ -121,46 +121,44 @@ export const uploadingAssets = count => ({
   count,
 });
 
-export const uploadAssetSuccess = (asset, response) => ({
+export const uploadAssetSuccess = response => ({
```

Comment:
The uploadAssets action creator has been modified to just fire an action with only the JSONified response as the payload, which includes the value of asset, so I've changed this action creator to just take response as a parameter. The reducer that consumes this action can pick out the asset from the response value.